topostats/default_config.yaml

```diff
@@ -33,14 +33,16 @@ grains:
   # Thresholding by area
   absolute_smallest_grain_size: 100
   absolute_area_threshold:
-    upper: [ 500, 800 ] # above surface [Low, High] in nm^2 (also takes null)
+    upper: [ 500, null ] # above surface [Low, High] in nm^2 (also takes null)
```

Comment:
Surprised this doesn't cause problems with some of the `minicircle` tests. I've certainly been bitten by having the wrong values here when testing my code.

I think generally we need to decide on a fixed set of values for the `default_config.yaml` and stick with them. If tests require them to be different values then the values in the config dictionaries can be changed in the tests.

I did create an issue for discussing and recording what the default values should be but can't find it right now.

Reply:
Just to update I found the issue I was thinking of, its #302.

Reply:
Yep this is my bad. There was no issue because the larger of the thresholds while it does pick up a large grain, it resides on the edge so is removed. So, more of a "lucky" mistake